android/app/src/main/java/com/zulipmobile/notifications/MessageInfo.java

```diff
@@ -0,0 +1,27 @@
+package com.zulipmobile.notifications;
+
+public class MessageInfo {
+    private String content;
+    private int messageId;
+
+    public MessageInfo(String content, int messageId) {
+        this.content = content;
+        this.messageId = messageId;
+    }
+
+    public String getContent() {
+        return content;
```

Comment:
Man, this makes me wish for [Kotlin](https://kotlinlang.org/docs/reference/data-classes.html) or something. :stuck_out_tongue:

```
data class MessageInfo(val content: String, val messageId: int)
```
96% fewer lines.

In fact, writing that out makes me think: do you want those setters? I think we actually don't -- I'd rather these things be immutable. That makes things easier to reason about.

(You don't seem to currently use the setters, happily.)

Reply:
Yeah, I wonder if kotlin is possible with react native, never researched about this.